dvc/repo/stage.py

```diff
@@ -502,3 +502,104 @@ def is_out_or_ignored(root, directory):
 
     def collect_repo(self, onerror: Callable[[str, Exception], None] = None):
         return list(self._collect_repo(onerror))
+
+    def _load_file(self, path):
```

Comment:
Duplicated from `load_file` above that also provides params/metrics/plots. Will refactor  in successive PRs. This method serves `_collect_all_from_repo` below.